### PR TITLE
Qualys_Add_New_Commands

### PR DESCRIPTION
**Pull request by DaniSalcedoGFT**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
None

## Description
We added five new commands to Qualys due to new requirements of our project that needed more functionality of existing 
functionality.
This includes functionality regarding of qualys-update-unix-record and new functionality in order to map ESXi actives against the VCenters

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by adi88d** [comment]
please sign our [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=31463) before we can accept your contribution.

File: Packs/qualys/Integrations/Qualysv2/Qualysv2.py [hunk 1]
```diff
@@ -4,11 +4,13 @@
 
 
 import requests
+
```
Comment (adi88d) [inline Packs/qualys/Integrations/Qualysv2/Qualysv2.py:7]: remove

File: Packs/qualys/Integrations/Qualysv2/Qualysv2.py [hunk 2]
```diff
@@ -4,11 +4,13 @@
 
 
 import requests
+				  
 
 from urllib3 import disable_warnings
 
 
 disable_warnings()  # pylint: disable=no-member
+
```
Comment (adi88d) [inline Packs/qualys/Integrations/Qualysv2/Qualysv2.py:13]: remove

File: Packs/qualys/Integrations/Qualysv2/Qualysv2.py [hunk 3]
```diff
@@ -1334,7 +1416,12 @@
             "end_after",
             "target_from",
             "tag_include_selector", "tag_exclude_selector", "tag_set_by", "tag_set_include", "tag_set_exclude",
+								   
+						 
+							  
+
```
Comment (adi88d) [inline Packs/qualys/Integrations/Qualysv2/Qualysv2.py:1422]: remove

**Review by adi88d — CHANGES_REQUESTED** [review]
Hi @DaniSalcedoGFT and thank you for your contribution!
- Please remove the empty lines from `Qualysv2.py`.
- Update release notes by using [demisto-sdk update-release-notes](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/update_release_notes/README.md) command.
- please format the `Qualysv2.yml` file and keep only the command you added.
- add the new commands to the `README.md` file by using [demisto-sdk generate-docs](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/generate_docs/README.md) command.

**Comment by adi88d** [comment]
Hi @DaniSalcedoGFT,
I see that you didnt update the `Qualysv2.py` file structure, Do you need any help with the pull request?

Please feel free to reach out to me here or on Slack.
Thanks again for contributing to our repo, hope to hear from you soon.

**Comment by DaniSalcedoGFT** [comment]
Hi @adi88d,
I'm short of time these days but I expect to push the changes before tomorrow ends.
Thanks for asking

**Comment by adi88d** [comment]
Your contribution was impacted by changes in the git history of the content repo, and you see many file changes that are not related to your PR. Our team is currently working on a solution, and we hope that it will be resolved soon.

**Comment by adi88d** [comment]
Hello @DaniSalcedoGFT  ,

Last Thursday, December 20th, there was an update in the content repo. This is why your PR looks like there were a lot of commits and files changed.
To fix this issue please do the following:
1. Save all of your changes locally or in a new branch.
2. Enter your forked repository page in GitHub.
3. At the top of the page, you will probably see something like this, please ensure you are in the master branch.
image

5. Click on “Sync fork” button on the top right side of the page.
image

6. The following message will appear (This branch has conflicts that must be resolved).

7. Click on the button “Discard commits”. Note that for this action to work, you should have permission to do a force push in the forked repo. If you don’t have one, ask the forked repo Owner to do that.
8. Enter your branch and repeat the same steps (4-6).
9. Check your PR, and see if your changes have been saved. If yes - you finished the update process.
10. If your PR does not show your changes, open a new PR with the changes that you saved at the beginning of the process.
11. In case you opened a new PR please close the old one, and update your reviewer with the new PR link.
12. Please paste on the new open PR the URL to your previous PR so our team member can continue the review from the same place.

Thank you very much for your contribution and your understanding of the situation.
Have a nice holiday season!

**Comment by adi88d** [comment]
Hi @DaniSalcedoGFT , 
Thank you for your contribution. 
I have not received a response from you in the last two weeks, so I will close your PR now.
Please feel free to re-open it when you are available to continue. 
Thanks again.

**Comment by DaniSalcedoGFT** [comment]
Hi @adi88d, I opened a new PR https://github.com/demisto/content/pull/31917
The new update on the content repo broke the push and every time I tried to push the content I wasn't pushing my changes

**Pull request closed without merging**